CHANGELOG.md

```diff
@@ -2,6 +2,25 @@
 
 ## Master (unreleased)
 
+### Breaking Changes
+
+* Removed `type` param from `status_tag` and some CSS classes [#4989][] by [@javierjulio][]
+  * The method signature has changed from:
+    ```ruby
+    status_tag(status, :ok, class: 'completed', label: 'on')
+    # now changes to:
+    status_tag(status, class: 'completed', label: 'on')
```

Comment:
Yes it can be added to demonstrate that's what it was originally doing and to maintain the same behavior it would have to be added if they were still using that CSS class.